docker/vnc/Dockerfile

```diff
@@ -54,20 +53,16 @@ RUN curl -sSfL https://sourceforge.net/projects/tigervnc/files/stable/${tigervnc
 
 #################################################################
 # Add desktop icons
+# Add wallpaper and force the default to point to the renku one
 
 COPY Git-GUI.desktop /home/jovyan/Desktop/
-RUN chmod +x /home/jovyan/Desktop/Git-GUI.desktop
-
 COPY gitk.desktop /home/jovyan/Desktop/
 COPY .git_icon.png /home/jovyan/Desktop/
-RUN chmod +x /home/jovyan/Desktop/gitk.desktop
-
-#################################################################
-# Add wallpaper and force the default to point to the renku one
-
-RUN mkdir -p /usr/share/backgrounds/renku/
 COPY renku_background_dots.png /usr/share/backgrounds/renku/
-RUN ln -s -f /usr/share/backgrounds/renku/renku_background_dots.png /usr/share/backgrounds/xfce/xfce-stripes.png
+RUN chmod +x /home/jovyan/Desktop/gitk.desktop && \
```

Comment:
No real changes - just reorganized commands so that we merge them into one large RUN command instead of having separate ones.